src/app/public/modules/radio/radio.component.ts

```diff
@@ -158,6 +212,9 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     }
   }
 
+/**
+ * Fires when users select a radio button.
+ */
   @Output()
   public get change(): Observable<SkyRadioChange> {
```

Comment:
We should probably document the properties for `SkyRadioChange`.
```
export interface SkyRadioChange {

  /**
   * Specifies the radio component that triggered the change event.
   * @internal
   */
  source: SkyRadioComponent;

  /**
   * Specifies the value from the radio component.
   */
  value: any;
}
```

Notice I marked the `source` as internal. We don't expose our components (`SkyRadioComponent` in this case) to consumers ever since version 4.0, so I don't think we'd want to document this property.

Reply:
Done!